packages/discord.js/typings/index.d.ts

```diff
@@ -4408,6 +4408,11 @@ export class GuildMemberRoleManager extends DataManager<Snowflake, Role, RoleRes
     roleOrRoles: RoleResolvable | readonly RoleResolvable[] | ReadonlyCollection<Snowflake, Role>,
     reason?: string,
   ): Promise<GuildMember>;
+  public modify(
+    rolesToAdd: readonly RoleResolvable[] | ReadonlyCollection<Snowflake, Role>,
+    rolesToRemove: readonly RoleResolvable[] | ReadonlyCollection<Snowflake, Role>,
+    reason?: any
```

Comment:
Add a comma for formatting.

```suggestion
    reason?: any, 
```